.github/workflows/release-build-publish.yml

```diff
@@ -55,11 +55,23 @@ jobs:
       - name: Build with Gradle
         run: ./gradlew clean build :p8e-api:jacocoTestReport --refresh-dependencies -Pversion=$VERSION
 
-      - name: Publish to GitHub Packages
-        run: ./gradlew publish -Pversion=$VERSION
+      - name: Install gpg secret key
+        run: |
+          export GPG_TTY=$(tty)
+          echo -n "${{ secrets.OSSRH_GPG_SECRET_KEY }}" | base64 --decode | gpg --batch --import
+          gpg --list-secret-keys --keyid-format LONG
+          echo -n "${{ secrets.OSSRH_GPG_SECRET_KEY }}" | base64 --decode > $GITHUB_WORKSPACE/release.gpg
+        if: github.event_name == 'release'
+
+      - name: Publish to Maven Central
+        run: |
+          ./gradlew publish \
+            -Pversion=$VERSION \
+            -Psigning.keyId=B7D30ABE -Psigning.password="${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}" -Psigning.secretKeyRingFile=$GITHUB_WORKSPACE/release.gpg \
+            --info
         env:
-          GITHUB_ACTOR: ${{ github.actor }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
```

Comment:
It appears these are all repository secrets currently, are you planning on making them organization-level secrets now that you have them all figured out so we can publish from other provenance-io repositories?

Reply:
Yes, the next time we add something that needs them I think that would make more sense.